src/imp/tracer_imp.js

```diff
@@ -680,6 +680,8 @@ export default class TracerImp extends EventEmitter {
     }
 
     _buffersAreEmpty() {
+        console.log("_buffersAreEmpty runtimeGUID", this._runtimeGUID);
```

Comment:
before my change, the guids here and at line 749 were different for the failing test case :-/